src/eddymotion/model/dipy.py

```diff
@@ -351,9 +332,58 @@ def compute_spherical_covariance(
     return np.where(theta <= a, 1 - 3 * (theta / a) ** 2 + 2 * (theta / a) ** 3, 0)
```

Comment:
This should be:

$$
        C(\theta) = \begin{cases}
            1 - \frac{3 \theta}{2 a} + \frac{\theta^3}{2 a^3} & \textnormal{if} \; \theta \leq a \\
            0 & \textnormal{if} \; \theta > a
        \end{cases}
$$

```suggestion
    return np.where(theta <= a, 1 - 3 * theta / (2 * a) + 2 * (theta ** 3) / (2 * (a ** 3)), 0)
```

Reply:
The plot in the notebook should give us an idea of whether the function is correct:
https://github.com/nipreps/eddymotion/blob/main/docs/notebooks/dmri_covariance.ipynb

Even if I do not see it with the above expression either :upside_down_face:.

Reply:
I'm getting this:

![covariance-plot](https://github.com/nipreps/eddymotion/assets/598470/ff7a9a86-31cd-448e-b65e-c942c5d25c0f)

We seem to have a factor of 2 somewhere, but I presume that comes from the b-value equalization.

Reply:
(notebook updated on https://github.com/nipreps/eddymotion/pull/216/commits/c05dfc05f152d6254bc4a68a7c361f0870c33414)